client/extensions/woocommerce/app/dashboard/required-plugins-install-view.js

```diff
@@ -106,6 +111,7 @@ class RequiredPluginsInstallView extends Component {
 	}
 
 	installPlugins = ( plugins ) => {
+		this.cancelUpdateTimeout();
 		const { site, wporg } = this.props;
 		for ( let i = 0; i < requiredPlugins.length; i++ ) {
```

Comment:
agreed, this whole thing needs to be rewritten once the calypso redux code is brought up to speed.  Didn't want that to be a dependency for our launch tho...